index.html

```diff
@@ -388,7 +426,49 @@ <h4 class="pageSubtitle">
                     {{starttime}}-{{endtime}}
                   </p>
                 </div>
-                <div class="{{#if twosessions}}col-md-5{{else}}col-md-10{{/if}}">
+                <div class="col-md-10 sessionlist">
```

Comment:
md is QUITE small in later versions of bootstrap. we may want to revise to lg?

Reply:
I think this part of the code isn't even used? In any event, there are many styling decisions that we're free to revisit. Let's just focus on making it work before we focus on making it look better.